presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -79,6 +79,7 @@
     private static final String TEMPORARY_STAGING_DIRECTORY_ENABLED = "temporary_staging_directory_enabled";
     private static final String TEMPORARY_STAGING_DIRECTORY_PATH = "temporary_staging_directory_path";
     private static final String PRELOAD_SPLITS_FOR_GROUPED_EXECUTION = "preload_splits_for_grouped_execution";
+    static final String WRITING_STAGING_FILE_ENABLED = "writing_staging_file_enabled";
```

Comment:
Make it public. Actually we should make all the properties here to be public.

Reply:
@arhimondr : Why is that? Shouldn't the actual access done through these `isXXX` static functions?

Reply:
Yes, we should. But it is useful to have the constant public, as you can use them in tests (when setting the test properties into a map)